pykern/pkcli/ci.py

```diff
@@ -43,6 +43,9 @@ def check_prints():
     from pykern import pkio
     from pykern import pkcli
 
+    def _error(msg):
+        pkcli.command_error("check_prints: {}", msg)
```

Comment:
I doubt you'll like this but lmk what should be here. Some message that "this occured in check_prints" would be helfpul otherwise one is left scratching their head when CI fails like https://github.com/radiasoft/rsconf/actions/runs/3292642290/jobs/5428245294#step:3:370